src/internal/util/pipe.ts

```diff
@@ -1,6 +1,11 @@
 import { identity } from './identity';
 import { UnaryFunction } from '../types';
 
+/**
+ * pipe() can be called on one or more functions, each of which can take one argument ("UnaryFunction"), to get back
+ * a function that passes the result of each subsequent UnaryFunction to the next one.  It's an implementation of
+ * what is described at https://typeofnan.dev/how-to-use-pipe-the-pipeline-operator-in-javascript/
+ */
```

Comment:
This is great! Really though, I think we need to move the doc comment to where it's above the implementation. Otherwise, I'm not sure the docs app will pick it up properly.